consensus/polybft/state.go

```diff
@@ -239,8 +229,8 @@ var (
 	exitEventsBucket = []byte("exitEvent")
 	// bucket to store commitments
 	commitmentsBucket = []byte("commitments")
-	// bucket to store bundles
-	bundlesBucket = []byte("bundles")
+	// bucket to store state sync proofs
+	stateSyncProofsBucket = []byte("stateSyncProofs")
```

Comment:
Just curious:
are we ever going to remove `stateSyncProofs` from the db or we must keep them forever, because of verification, since we don't know when somebody can query for the proof even if the state sync is fully executed?

Reply:
We're going to leave them forever for that very reason.

Reply:
👌